collector/src/benchmark/mod.rs

```diff
@@ -183,11 +187,22 @@ impl Benchmark {
     ) -> anyhow::Result<()> {
         let iterations = iterations.unwrap_or(self.config.runs);
 
-        if self.config.disabled || profiles.is_empty() {
+        let profiles: Vec<Profile> = profiles
+            .into_iter()
+            .copied()
```

Comment:
nit: is the copied necessary?

Reply:
Well, it is for the return value to `Vec<Profile>`. It's not needed if it was `Vec<&Profile>`.

Reply:
Ah sorry - I thought the first profiles was a `Vec<Profile>` so `into_iter` yielded `Profile` not `&Profile`. Sorry for the noise!